.github/workflows/verify-test.yml

```diff
@@ -0,0 +1,72 @@
+# this workflow can be used to check if given integration test is flaky
+name: verify-test
+
+on:
+  workflow_dispatch:
+    inputs:
+      testName:
+        description: 'Test name'     
+        required: true
+      count:
+        description: 'Test execution count'     
+        default: '20'
+
+jobs:
+  verify-test:
+    strategy:
+      fail-fast: false
+      matrix:
+        machine: [ windows-2022, ubuntu-20.04, macos-11 ]
+    runs-on: ${{ matrix.machine }}
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - uses: actions/setup-dotnet@v2.1.0
+        with:
+          dotnet-version: | 
+            3.1.x
+            6.0.x
+
+      - name: Windows - build and test ${{ github.event.inputs.testName }} ${{ github.event.inputs.count }} times
+        run: |
+          ./build.cmd BuildTracer ManagedTests --skip RunManagedTests
+          if ($global:LASTEXITCODE -ne 0) { 
+            exit $global:LASTEXITCODE
+          }
+          
+          for ($i=1; $i -le ${{ github.event.inputs.count }}; $i++) {
```

Comment:
I am considering adding a `TestCount` nuke parameter instead.
What is your preference?

Reply:
It could be useful for local testing. I like the idea.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1229/commits/7a4284731aeca05dcea8710fc6184c62c699b6b6